pylib/cqlshlib/test/test_cqlsh_completion.py

```diff
@@ -1184,9 +1184,15 @@ def test_complete_in_alter_type(self):
         self.trycompletions('ALTER TYPE IF EXISTS new_type ADD IF NOT EXISTS ', choices=['<new_field_name>'])
         self.trycompletions('ALTER TYPE IF EXISTS new_type RENAME ', choices=['IF', '<quotedName>', '<identifier>'])
 
+    # USER checks
+    def test_complete_in_create_user(self):
+        self.trycompletions('CREATE USER ', choices=['<username>', 'IF'])
+        self.trycompletions('CREATE USER IF ', immediate='NOT EXISTS ')
```

Comment:
Good catch with the reordering. Not specifically related to the change for this PR tho. What do you think @frankgh ? Should we follow the [Boy Scout rule](https://biratkirat.medium.com/step-8-the-boy-scout-rule-robert-c-martin-uncle-bob-9ac839778385) here?